pennylane_lightning/core/src/measurements/tests/Test_MeasurementsBase.cpp

```diff
@@ -1257,6 +1257,7 @@ void testSamples(const std::optional<std::size_t> &seed = std::nullopt) {
         // This object attaches to the statevector allowing several
         // measurements.
         Measurements<StateVectorT> Measurer(statevector);
+        Measurer.setDeviceSeed(seed);
```

Comment:
Where is `seed` defined?

Reply:
It comes as an argument of the `testSamples` function

Reply:
https://github.com/PennyLaneAI/pennylane-lightning/blob/f419d371810400b7cb8c6f3b8cd7cf959beb595b/pennylane_lightning/core/src/measurements/tests/Test_MeasurementsBase.cpp#L1307